_backend/config.example.php

```diff
@@ -4,17 +4,17 @@
 
 return array(
   'release_title'    => 'Odin',
-  'release_version'  => '6',
-  'release_filename' => 'elementaryos-6.0-stable.20210901.iso',
-  'release_size'     => '2.49 GB',
-  'release_magnet'   => '73e9c0288c0b62c2646b695219b550fd231fede4',
-  'release_sha256'   => '439110ceadadd13683632ddd273095f72ee9b8d22a7569ac84c2fd1c4d920610',
+  'release_version'  => '6.0',
+  'release_filename' => 'elementaryos-6.0-stable.20210810.iso',
+  'release_size'     => '2.19 GB',
```

Comment:
I have to double-check this, but I recall seeing 2.4 GB everywhere from Files to Digital Ocean.

Reply:
Builds site also says 2.36 GB

```suggestion
  'release_size'     => '2.36 GB',
```

Reply:
My guess is the 2,19GB might come from some method similar to explorer, where it does some funky rounding. It *should* be 2,36GB
![image](https://user-images.githubusercontent.com/30969346/128891383-baa9ffc1-e2e9-4e22-8a81-0605e04d19eb.png)